qa/automation/CareerPortal.Tests.Scenarios/3. Send_CV/Send_CV.feature

```diff
@@ -0,0 +1,135 @@
+Feature: Send_CV
+
+Background:
+    Given user visited the site https://career.quantori.com/ru
+
```

Comment:
In order to Send CV  you need to choose any "Job position " in homepage.  But you start Send CV  from  homepage there is not any form fileds.

Reply:
ok but we test on https://qa-career.quantori.com/  not on prod